awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -53,12 +54,22 @@ def intrinsics_multi_constructor(loader, tag_prefix, node):
     return {cfntag: value}
 
 
+def str_representer(dumper, node):
+
+    if re.match(r'[-+]?[0-9][0-9_]*', node):
```

Comment:
I would rather quote all strings. If a value is parsed as string, it should be okay to quote them. What do you think?